test/syntax/json/uniontype.json

```diff
@@ -1,87 +1,127 @@
 [
-    {
-        "type": "interface",
-        "name": "Union",
-        "partial": false,
-        "members": [
-            {
-                "type": "attribute",
-                "static": false,
-                "stringifier": false,
-                "inherit": false,
-                "readonly": false,
-                "idlType": {
-                    "sequence": false,
-                    "generic": null,
-                    "nullable": false,
-                    "array": false,
-                    "union": true,
-                    "idlType": [
+   {
+      "partial" : false,
+      "members" : [
+         {
+            "idlType" : {
+               "idlType" : [
+                  {
+                     "array" : false,
```

Comment:
(eventually, we might want to generate this stuff using JSON.stringify() and we won't be able to match this format).